src/ray/gcs/client_test.cc

```diff
@@ -12,6 +12,12 @@ extern "C" {
 
 namespace ray {
 
+namespace gcs {
+
+namespace {
+const char *const kRandomId = "abcdefghijklmnopqrst";
```

Comment:
preferable to use `constexpr`.

Reply:
`const char* const` is actually _a_ preferred way to declare string constants.  Relevant snippets from 
https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables:
```
// bad: non-trivial destructor
const string kFoo = "foo";
```
`Global strings: if you require a global or static string constant, consider using a simple character array, or a char pointer to the first element of a string literal. String literals have static storage duration already and are usually sufficient.`

Reply:
The link seems to be fine with `constexpr`, which allows the compiler to optimize more than `const`.

Reply:
Changed to `constexpr char kRandomId[]`.  Which I confirm is the preferred way in TF.